pkgs/top-level/all-packages.nix

```diff
@@ -2229,6 +2229,8 @@ with pkgs;
 
   diff-so-fancy = callPackage ../applications/version-management/diff-so-fancy { };
 
+  gcli = callPackage ../applications/version-management/gcli { };
```

Comment:
The program should be on the `pkgs/by-name/so/some-example/package.nix` hierarchy, as it is the new preferred way of adding packages.
```suggestion
```

Reply:
Oh cool, I totally missed that!